src/server.js

```diff
@@ -29,12 +29,15 @@ app.use((err, req, res) => {
 
   // Return 500 error code
   res.status(500).send({ message: 'Error' });
+
+  next();
 });
 
 // Handle 404s
 app.use((req, res, next) => {
   res.status(404).send({
-    message: 'Houston, we\'ve had a problem here',
+    message: 'Houston, fix the problem',
     urlRequested: `${req.method} ${req.url}`,
   });
+  next();
```

Comment:
Same as above. Not sure if this function call should be here. I could be wrong though.